setup.py

```diff
@@ -69,6 +69,13 @@ def set_systemd_files(data_files, dest="/lib/systemd/system", src=None):
     data_files.append((dest, src))
 
 
+def set_systemd_cgroups_files(data_files, dest="/etc/systemd/system", src=None):
+    if src is None:
+        src = ["init/ubuntu/azure.slice",
```

Comment:
let's add these along with walinuxagent.service on line 160 and remove this function, which makes the distinction of "cgroups files". walinuxagent.service will also include parameters for cgroups so there is really no distinction between these 3 files